game/script-introduction.rpy

```diff
@@ -1,3 +1,5 @@
+default persistent.jn_player_profanity_during_introduction = False
```

Comment:
we should use the `_jn` prefix here. I'd like to move all our stuff to use it since really they are more or less semi-private considering usage.